pkg/apis/submariner.io/v1/types.go

```diff
@@ -344,3 +344,51 @@ type GlobalIngressIPList struct {
 
 	Items []GlobalIngressIP `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:shortName="gwrt"
+
+type GatewayRoute struct {
```

Comment:
Just thinking out loud, should we use OVN in the names of the struct? 
In the current state, this could give a false impression that this CR will be used in all types of deployments.

Reply:
I think we discussed in the proposal right? Like though this may not be used for any other CNI nows , we are not sure if there can be usecases later. Also this is going to be created only for OVN right now

Reply:
Yes we did, but I wanted to throw it open as this is very specific to OVN and not sure if we will use it for other CNIs. Anyways, its okay.